src/Sylius/Bundle/ReviewBundle/Resources/config/services.xml

```diff
@@ -27,7 +27,7 @@
         <service id="sylius.average_rating_calculator" class="Sylius\Component\Review\Calculator\AverageRatingCalculator" />
         <service id="sylius.average_rating_updater" class="Sylius\Bundle\ReviewBundle\Updater\AverageRatingUpdater">
             <argument type="service" id="sylius.average_rating_calculator" />
-            <argument type="service" id="doctrine.orm.default_entity_manager" />
+            <argument type="service" id="sylius.manager.product_review" />
```

Comment:
Thanks God, to have Travis CI, I have approved this changes, but I just have checked the failing build. And we cannot do this change, because it would couple ReviewBundle with ProductBundle....